setup.py

```diff
@@ -32,5 +32,5 @@ def readall(path):
         "Topic :: Utilities",
     ],
     python_requires=">=3.6",
-    install_requires=["python-docx"],
+    install_requires=["python-docx", "cssutils"],
```

Comment:
This package hasn't had a commit or release since 2017-04-14. It is only tested against Python 3.5. It looks unmaintained. Could we pick a a better maintained library?

Would https://github.com/Kozea/tinycss2 be suitable? I'm also open to other suggestions if you have them.